src/ray/protobuf/gcs_service.proto

```diff
@@ -98,6 +98,16 @@ message GetNamedActorInfoReply {
   ActorTableData actor_table_data = 2;
 }
 
+message GetAllActorNamesRequest {
+  // Namespace to filter to.
+  string ray_namespace = 2;
```

Comment:
Could we say this is empty for all namespaces?

Reply:
Unfortunately an empty namespace is valid...